include/tvm/relax/attrs/ccl.h

```diff
@@ -40,17 +40,21 @@ struct AllReduceAttrs : public tvm::AttrsNode<AllReduceAttrs> {
   }
 };  // struct AllReduceAttrs
 
-/*! \brief Attributes used in scatter_from_worker0 operators */
-struct ScatterFromWorker0Attrs : public tvm::AttrsNode<ScatterFromWorker0Attrs> {
+/*! \brief Attributes used in scatter operators */
+struct ScatterAttrs : public tvm::AttrsNode<ScatterAttrs> {
```

Comment:
Scatter is a bit ambiguous. It could mean an op or a communication collective. What do you have a better naming specifically for this usecase?

Reply:
Good point. I have no idea on this naming. @tqchen Do you have any suggestions? This attr is expected to be reused by R.dist.redistribute_replica_to_shard we discussed before.

Reply:
How about ScatterCollectiveAttr?

Reply:
sounds good